src/App/Application.cpp

```diff
@@ -101,6 +101,8 @@
 #include <Base/UnitsApi.h>
 #include <Base/VectorPy.h>
 
+#include <Gui/Navigation/NavigationStyle.h>

```

Comment:
is this supposed to be here?

Reply:
perhaps I'm missing something but I don't see you using anything from this header so it seems unnecessary, besides that App is not supposed to depend on Gui, although it's already been "tainted" (#20897)